.travis.yml

```diff
@@ -45,6 +45,10 @@ matrix:
     fast_finish: true
     include:
 
+        # Try with pint
+        - python: 3.6
```

Comment:
I think we should be testing anything we think users might be using. So I think 2.7 should be included. More generally, I think we should also be covering Python 3.5, since a lot of people might still be using that.